packages/jaeger-ui/src/components/SearchTracePage/SearchForm.js

```diff
@@ -38,9 +38,12 @@ import './SearchForm.css';
 const FormItem = Form.Item;
 const Option = Select.Option;
 
-const AdaptedInput = reduxFormFieldAdapter(Input);
-const AdaptedSelect = reduxFormFieldAdapter(Select);
-const AdaptedVirtualSelect = reduxFormFieldAdapter(VirtSelect, option => (option ? option.value : null));
+const AdaptedInput = reduxFormFieldAdapter({ AntInputComponent: Input, isValidatedInput: true });
```

Comment:
Would suggest breaking this out into two wrapped components, one with `isValidatedInput: true` and one without.

Reply:
yup! meant to do that all along. updated!